<launch>

    <!-- <param name="/use_sim_time" value="true" /> -->
    <arg name="reconfigure" default="false" />

    <node if="$(arg reconfigure)" name="configure" pkg="rqt_reconfigure" type="rqt_reconfigure"> </node>
    <node name="body_node" pkg="arm_control" type="body_node" respawn="true" output="screen"> </node>

</launch>
